engine/apps/aliyun_dyvms/phone_provider.py

```diff
@@ -0,0 +1,183 @@
+            phoneCall = AliyunDyvmsPhoneCall(
+                status=AliyunDyvmsCallStatuses.IN_PROCESS,
+                call_id=call_id,
+            )
+
+            record_call_status_async.delay(call_id)
+
+            return phoneCall
+        except ServerException as server_err:
+            logger.error(f"AliyunDyvmsPhoneProvider.make_notification_call: failed, number {number}, {server_err}")
+            raise FailedToMakeCall(graceful_msg=f"AliyunDyvmsPhoneProvider.make_notification_call: failed, number "
+                                                f"{number}, {server_err}")
+        except ClientException as client_err:
+            logger.error(f"AliyunDyvmsPhoneProvider.make_notification_call: failed, number {number} {client_err}")
+            raise FailedToMakeCall(graceful_msg=f"AliyunDyvmsPhoneProvider.make_notification_call: failed, number "
+                                                f"{number}, {client_err}")
+        except Exception as e:
+            logger.error(f"AliyunDyvmsPhoneProvider.make_notification_call: failed, number {number} {e}")
+            raise FailedToMakeCall(graceful_msg=f"AliyunDyvmsPhoneProvider.make_notification_call: failed, number "
+                                                f"{number}, {e}")
+
+    def make_call(self, number: str, message: str):
+        logger.info(f"AliyunDyvmsPhoneProvider.make_call: start")
+        try:
+            formated_message = f'{{"title":"{message}"}}'
+            response = self._call_create(live_settings.ALIYUN_DYVMS_TTS_CODE, number, formated_message)
+            acs_response_from_json: AcsResponse = AcsResponse.from_json(response)
+            if acs_response_from_json.Code != "OK":
+                logger.error(f'AliyunDyvmsPhoneProvider.make_notification_call to {number}: failed'
+                             f'{acs_response_from_json.Code}, {acs_response_from_json.Message}')
+                raise FailedToMakeCall(graceful_msg=f'Failed make notification call to {number},'
+                                                    f'{acs_response_from_json.Code}, {acs_response_from_json.Message}')
+            call_id = acs_response_from_json.CallId
+
+            if not call_id:
+                logger.error("AliyunDyvmsPhoneProvider.make_notification_call: failed, missing call id")
+                raise FailedToMakeCall(graceful_msg=f'Failed make notification call to {number}, missing call id'
+                                                    f'{acs_response_from_json.RequestId}')
+
+            logger.info(f"AliyunDyvmsPhoneProvider.make_notification_call: success, call_id {call_id}")
+        except ServerException as server_err:
+            logger.error(f"AliyunDyvmsPhoneProvider.make_notification_call: failed, number {number}, {server_err}")
+            raise FailedToMakeCall(graceful_msg=f"AliyunDyvmsPhoneProvider.make_notification_call: failed, number "
```

Comment:
I propose to be more accurate with graceful_msg. If it's present it will be shown to user in a certain scenarios, so it shouldn't containt messages which shouldn't be exposed to user.  I encourage you to take a look on TwilioPhoneProvider.__get_graceful_msg method as an example. It processes error codes from Twilio and converts to user-visible errors or just returns none when details of the error should be hidden from user.